PROJECT

```diff
@@ -22,4 +22,13 @@ resources:
   kind: Store
   path: github.com/deislabs/ratify/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: ratify.deislabs.io
+  group: config
+  kind: CertificateStore
+  path: github.com/deislabs/ratify/api/v1alpha1
+  version: v1alpha1
```

Comment:
just curious, what do these versions correspond to?

Reply:
discussed offline, we should check with Anish to see if we need to pump up these version Post RC1